cpp/server/ct-dns-server.cc

```diff
@@ -22,7 +22,7 @@ DEFINE_string(domain, "", "Domain");
 DEFINE_string(db, "", "Database for certificate and tree storage");
 
 // Basic sanity checks on flag values.
-static bool ValidatePort(const char* flagname, int32_t port) {
+static bool ValidatePort(const char*, int32_t port) {
```

Comment:
You know, now that I think about it, saying "this port is invalid" without telling them *which* port, that's a bit rude, especially if we know which port! :smiley: